### DISCUSSION TITLE
Calculators! 🙂

### DISCUSSION THREAD

**Discussion by GalaxyM105** [Darkwater23/MeadMate#104] [Ideas]
Hello! 

I just finished my first 1 gallon batch of mead and I am trying my best to research everything about mead making to make my meads the best I can. I am on the r/mead subreddit and am loving talking to the community. I think I saw a post that you made on r/mead a few years ago that was deleted. Are you still active in r/mead?

Anyway, I was wondering if you had any plans on adding a lot of calculators to your app? 🙂 I know that there is almost an exhaustive list of calculators that would be beneficial for mead makers. A few calculators that I know of are:

- Wort Factor Correction Calculator for Calibrating a Refractometer
- Hydrometer Calculator for adjusting for temperature
- Mead Batch Calculator for figuring out what and how much of different types of variables are needed to make a dry, semi-sweet, and sweet mead of a certain volume. Including yeast nutrients and steps to implement.

I know that there are already calculators available on the web for these things, but it would be awesome to just have them all in one app!  I'm not much of a programmer now a days. I wasn't that great at it in college, but when I have time I am trying to brush up on learning Javascript and have thought about looking into frameworks like React. 

I'm excited that this is an open source project and seeing where it goes. 🙂 Have you advertised it more recently on reddit and elsewhere?

cheers!

**Comment by Darkwater23** [comment]
Extra calculators could be a nice addition to the app! I'll add these to the list.

This is my first open source project. I don't get PRs, though, because I slapped this together with Mobile jQuery. One of these days I'll learn how to do a mobile app properly and upgrade the project.
